cmis/fields.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from operator import attrgetter
+from openerp import fields
+from openerp.exceptions import UserError
+
+
+class CmisFolder(fields.Field):
+    """ A reference to a cmis:folder. The reference must be formatted as
+    follow: backend_name + ':' cmis:objectId
+    
+    :param backend_name:
+    
+    The attribute ``backend_name`` is mandatory
+    
+    :param allow_create: (by default True)
+    
+    :param allow_delete: (by default False)
+    
+    :param cmis_create_method:
+    
+    :param cmis_path: (by default backend.initial_directory_write + '/' model._name)
+    
+    :param cmis_name_get: (by default instance.name_get)
+
+    """
+    type = 'char'  # Postgresl
+    widget = 'cmis_folder'  # Web widget
+    _slots = {
+        'backend_name': None,
+        'cmis_name_get': 'name_get',
+        'allow_create': True,
+        'allow_delete': False,
+        'cmis_path': None,
+        'cmis_create_method': None
```

Comment:
create_method ?

If create_root_get and create_name_get are method, this will rarely be used, but could be useful nevertheless.